lib/utils.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+const bytes = require('bytes');
```

Comment:
although it is nice to be able to add with human readable sizes, I think this can be used by the user on top of Bull, just to keep the dependencies as low as possible.

Reply:
Sounds good. So I'll remove this dependency and if the user wants to add something like that to pass it in, they can.